src/services/application/loginService.ts

```diff
@@ -23,9 +23,8 @@ export const loginService = async (
     } catch (e) {
       console.log(e);
```

Comment:
A console.error() will make this more visible

Reply:
We could, but we will later on add an error handler connected to application insights. We just left this here but it was not really related to this task. But we can change it to console error if you fell more comfortable with it 😄